launcher/minecraft/skins/SkinModel.cpp

```diff
@@ -110,7 +116,7 @@ SkinModel::SkinModel(QDir skinDir, QJsonObject obj)
         m_model = Model::SLIM;
     }
     m_path = skinDir.absoluteFilePath(name) + ".png";
-    m_texture = QImage(getSkin(m_path));
+    m_texture = getSkin(m_path);
```

Comment:
Indenting mistake? Not sure why this isn't picked up....

Reply:
I don't see a problem here either...